docs/development/development.md

```diff
@@ -0,0 +1,42 @@
+# Development
+
+We will describe everything related to getting started with development on tfchain here.
+
+## Setup work environment
+
+See [setup](./setup.md)
+
+## Get the code
+
+```
+git clone git@github.com:threefoldtech/tfchain.git
+```
+
+## Installing and running tfchain in dev mode
+
+```
+cd tfchain/substrate-node
+cargo build
+
+./target/debug/tfchain --dev --ws-external --pruning archive
+```
+
+## Running multi node local tfchain network
```

Comment:
Please give a brief description about this section before linking to the local multinode file